cmd/cli/kubectl-kyverno/commands/test/command.go

```diff
@@ -158,8 +158,12 @@ func checkResult(test v1alpha1.TestResult, fs billy.Filesystem, resoucePath stri
 		expected = test.Status
 	}
 	// fallback on deprecated field
+	patchedResource := test.PatchedResources
 	if test.PatchedResource != "" {
-		equals, err := getAndCompareResource([]*unstructured.Unstructured{&response.PatchedResource}, fs, filepath.Join(resoucePath, test.PatchedResource))
+		patchedResource = test.PatchedResource
+	}
```

Comment:
In this case if both new and deprecated field are specified, the deprecated field will overwrite the non deprecated field, which is wrong imo

it should be the other way around

Reply:
Done.